packages/orbit-components/src/Drawer/index.tsx

```diff
@@ -40,46 +40,36 @@ const Drawer = ({
   ariaLabel,
 }: Props) => {
   const overlayRef = React.useRef(null);
-  const closeButtonRef = React.useRef<HTMLButtonElement | null>(null);
-  const scrollableRef = React.useRef<HTMLElement | null>(null);
+  const drawerRef = React.useRef<HTMLElement | null>(null);
 
   const [overlayShown, setOverlayShown, setOverlayShownWithTimeout] = useStateWithTimeout<boolean>(
     shown,
     parseFloat(theme.orbit.durationNormal) * 1000,
   );
 
-  const handleOnClose = React.useCallback(
-    (ev: React.MouseEvent<HTMLDivElement>) => {
-      if (onClose && ev.target === overlayRef.current) {
-        onClose();
-      }
-    },
-    [onClose],
-  );
-
-  useFocusTrap(scrollableRef);
-  useLockScrolling(scrollableRef, lockScrolling && overlayShown);
-
-  const handleKeyDown = React.useCallback(
-    (ev: React.KeyboardEvent<HTMLDivElement> | KeyboardEvent) => {
-      if (ev.keyCode === KEY_CODE_MAP.ESC && onClose) {
-        onClose();
-      }
-    },
-    [onClose],
-  );
+  useFocusTrap(drawerRef);
+  useLockScrolling(drawerRef, lockScrolling && overlayShown);
```

Comment:
When `lockScrolling=true` and I scroll down, the drawer doesn't have full height. That's not happening on production.
![image](https://github.com/user-attachments/assets/32e6777e-086b-46b3-a461-215259a2445f)

Reply:
Good catch. Changed the positioning